src/structures/GuildChannel.js

```diff
@@ -539,6 +539,17 @@ class GuildChannel extends Channel {
     return this.permissionsFor(this.client.user).has(Permissions.FLAGS.MANAGE_CHANNELS);
   }
 
+  /**
+ * Whether the channel is manageable by the client user
+ * @type {boolean}
+ * @readonly
+ */
+  get manageable() {
+    if (this.client.user.id === this.guild.ownerID) return true;
+    if (this.permissionsFor(this.client.user).has([Permissions.FLAGS.MANAGE_CHANNELS, Permissions.FLAGS.VIEW_CHANNEL])) return true;
```

Comment:
You can just return this permissions check rather than using a conditional.

Reply:
I’ll check lint errors and test runs when I get home. I thought could process Space’s requested changes from mobile but apparently not.